.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
+
+
+# commands to install dependencies
+install:
+  - pip install -r requirements.txt
```

Comment:
we also need to `pip install -r requirements-e.txt` because discord.py is in there